### DISCUSSION TITLE
How can I show pdf files from assets?

### DISCUSSION THREAD

**Discussion by ghost** [afreakyelf/Pdf-Viewer#6] [Q&A]
I see that this library asks for pdfURL. I cant find a way to convert assets path to URL. So how can i render pdf file stored in asset directory?

**Comment by afreakyelf** [comment]
Hi Devesh, thanks for checking out. As of now i haven’t added support for local files. Feel free to contribute or give me few days, I will add.

**Reply by ghost** [reply]
Thanks Rajat for the reply. I am eagerly waiting for the feature to be incorporated.

**Comment by afreakyelf** [comment]
Hi @deveshdron , as per your request I have added support for files in assets folder. Kindly checkout my latest release (v1.0.4).
Don't forget to ⭐.

🍻
